Big-Spender/readme.md

```diff
@@ -68,39 +69,48 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT : select transaction_no, description from spends where lower(description) like '%fee%';
```

Comment:
Yep, looks good. 

What about` ilike`? for case in-sensitivity rather than just lower? (although this also works ok!)